lib/index.js

```diff
@@ -59,6 +60,10 @@ function shelfLibFactory(hostPrefix, libOptions) {
         libOptions.timeoutDuration = 300000;
     }
 
+    if (typeof libOptions.retries === "undefined") {
```

Comment:
We do care if retries are 0. I could specifically check for null or false if you'd like